internal/printer/printer.go

```diff
@@ -141,6 +145,10 @@ func (p *printer) printCSSImports(cssLen int) {
 		p.print(fmt.Sprintf("import \"%s?astro&type=style&index=%v&lang.css\";", p.opts.Filename, i))
 		i++
 	}
+	if p.needsTransitionCSS {
+		p.addNilSourceMapping()
+		p.print(`import "astro/components/viewtransitions.css";`)
```

Comment:
Should we add a new experimental option for this instead? It would also help with conditionally importing `renderTransition` so we don't break older versions of Astro that uses the latest compiler version, as they don't export `renderTransition`

Reply:
Ah yes, I didn't think about that, but good point.